fetch/api/request/destination/fetch-destination-style.https.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../../../../service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<script>
+async_test(function(t) {
+    var worker_url = 'resources/fetch-destination-worker.js';
+    var scope = 'resources/empty.html';
+    var registration;
+
+    service_worker_unregister_and_register(t, worker_url, scope)
+      .then(t.step_func(function(r) {
+          registration = r;
+          return wait_for_state(t, r.installing, 'activated');
+        }))
+      .then(t.step_func(function() {
+          return with_iframe(scope);
+        }))
+      .then(t.step_func(function(frame) {
+          var link = frame.contentWindow.document.createElement("link");
+          link.rel = "stylesheet";
+          link.addEventListener("load", t.step_func_done(function() { registration.unregister(); }));
+          link.addEventListener("error", t.step_func_done(function() {
+              registration.unregister();
+              assert_unreached("Fetch destination preload errored");
+          }));
+          link.href = "resources/dummy?dest=style";
+          frame.contentWindow.document.body.appendChild(script);
```

Comment:
I'm assuming you meant to append 'link' here, not 'script'?

Reply:
Yeah, in the latest version of the tests, it's all just `node`